### ISSUE TITLE
Switch to PySide2

### ISSUE THREAD

**Issue by JayFoxRox** [bartoszek/AUR-luxcorerender#1]
This should be supported in the latest release. See https://github.com/LuxCoreRender/LuxCore/pull/170

pyside2 and pyside2-tools are in "extra" repositories; meaning a lot of AUR make-deps can probably be avoided.

**Comment by bartoszek** [comment]
Thanks for the tip, gave me a moment to push the changes :smirk:

**Comment by JayFoxRox** [comment]
Thanks for the quick response!

Apparently the original PySide2 PR on the upstream was very broken.
I'm feeling a bit guilty about this, as I came here without testing the changes myself (trusting the upstream contributors / maintainers in the process). I'll try to fix the situation as soon as possible.

I've already created 2 reports upstream about this:

- https://github.com/LuxCoreRender/LuxCore/issues/192 (Issue; about broken PySide2)
- https://github.com/LuxCoreRender/LuxCore/issues/189 (PR; about broken PySide and broken PySide2)

However, it's not worth reverting the AUR switch to PySide2 probably, considering nobody has noticed these issues in 4 months and the fixes are already on their way.
